### DISCUSSION TITLE
Dependencies like Visual C++ in one single package

### DISCUSSION THREAD

**Discussion by KaranKad** [microsoft/winget-cli#610] [General] [answered]
It would be great to have all Visual C++ dependencies from old to new ones in one single package and have them updated through windows updates or MS Store.
Dependencies are represented separately and and are not preinstalled on windows. When you install app using winget dependencies are not installed automatically hence the app does not work.
![image](https://user-images.githubusercontent.com/71691514/96142188-6d794900-0f1f-11eb-9041-5013cd0db9fc.png)
Having all dependencies in one package will help winget.
I would like to know if this is possible.

**Comment by denelon** [comment, marked as answer]
We're planning on implementing dependency support. They will be identified in the manifest so the package manager can install them ahead of the package.
